src/main/resources/swagger/api-docs.yaml

```diff
@@ -1481,18 +1621,6 @@ paths:
           description: Submission or Workflow not found
         500:
           description: Internal Error
-  /api/submissions/queueStatus:
```

Comment:
due to previous copy-paste error, this queueStatus endpoint was in the wrong place. I relocated it to maintain alphabetization and updated its contents.